includes/modules/order_total/ot_tax.php

```diff
@@ -46,22 +49,22 @@ function process()
                 unset($order->info['tax_groups'][0]);
             }
             foreach ($order->info['tax_groups'] as $key => $value) {
-                if (SHOW_SPLIT_TAX_CHECKOUT == 'true') {
-                    if ($value > 0 or ($value == 0 && STORE_TAX_DISPLAY_STATUS == 1)) {
+                if (SHOW_SPLIT_TAX_CHECKOUT === 'true') {
+                    if ($value > 0 || ($value === 0 && STORE_TAX_DISPLAY_STATUS === '1')) {
```

Comment:
Since the $value is a float, this is a case to use kinda-sorta-equal-to instead of a strict comparison.  Here and on line 61.

Reply:
good point!.